src/components/AuthRequired.js

```diff
@@ -1,91 +1,100 @@
 import React, { Component } from 'react'
 import { withRouter } from 'react-router'
-import { loginInit, loginSuccess, loginError } from '../store/actions/auth.actions'
+import {
+  loginInit,
+  loginSuccess,
+  loginError,
+} from '../store/actions/auth.actions'
```

Comment:
Why it looks this way? :O

Reply:
@ulaulala I believe it was well described there
https://prettier.io/docs/en/option-philosophy.html

it is about philosophy they believe around endless debate over styling;

1) The + I really see is github's diff, where you can exactly see which named import was added, instead diff'ing the whole line; its similar to object properties being multiline instead one-line.
2) Not sure from which "styleguide" they took it exactly but seems that, among many others, airbnb uses similar multiline-import rule.
3) They point out as well that it is consistent with all others `curly braces blocks` - which also applies to 1)

More:
https://github.com/airbnb/javascript#modules--multiline-imports-over-newlines

Reply:
1. Github always highlight the words which were added in the line :) So you don't need to wonder about it, github tell you about changes in this line by highlighting line with lightgreen color, and changed/added words by darker one
2. Why we should use the same styleguide as Airbnb?
3. Not all curly braces blocks are multiline, e.g. destructed objects (like in line 50 in this file after your changes). In my opinion if the names of properties are short, we should leave it in one line. It always depends on something, so I vote on no-restrictions about single/multiline imports. Alternatively we can specify that imports with more than X items inside curly braces should be multiline. But I think X should be bigger than 3 for sure.
This import is still goodlooking in one line:
`import { apple, banana, lemon, berry, orange } from 'fruits'`
but this is not:
`import { authorizationAccessToken, cookiesConfirmation, requestErrorMessage } from 'important-constants'`